torch/_inductor/fx_passes/fuse_attention.py

```diff
@@ -259,6 +259,27 @@ def _sfdp_replacement_10(query, key, value):
     )
 
 
+def _sfdp_pattern_11(query, key, value, inv_scale):
```

Comment:
Can you add a dropout version of this as well ? We have been registering dropout and non-dropout versions of each pattern.

Reply:
Thanks. I have added the dropout version.